packages/store/src/state-factory.ts

```diff
@@ -2,18 +2,40 @@ import { Injector, Injectable, SkipSelf, Optional } from '@angular/core';
 import { Observable, of, forkJoin, from } from 'rxjs';
 import { shareReplay, takeUntil, map } from 'rxjs/operators';
 
-import { META_KEY, StateContext, ActionOptions, NgxsLifeCycle } from './symbols';
-import { topologicalSort, buildGraph, findFullParentPath, nameToState, MetaDataModel, isObject } from './internals';
+import { META_KEY, StateContext, NgxsLifeCycle } from './symbols';
+import {
+  topologicalSort,
+  buildGraph,
+  findFullParentPath,
+  nameToState,
+  isObject,
+  ActionHandlerMetaData,
+  ObjectKeyMap,
+  StateKlass,
+  GetStateFn,
+  SetStateFn,
+  DispatchFn
+} from './internals';
 import { getActionTypeFromInstance, setValue, getValue } from './utils';
 import { ofActionDispatched } from './of-action';
+import { Actions } from '@ngxs/store';
+import { InternalActions } from '@ngxs/store/src/actions-stream';
+
+interface MappedStore {
```

Comment:
Typically I like to put these in a seperate file, i think symbols is what we been using.

Reply:
I kept this local to the file because it is not needed elsewhere and therefore not exported. Let me know what way you want me to go with this... keep it here or move it to another file `Symbols.ts` or `internal.ts`.

Reply:
I'd say they are fine here. But Austin can make the final call

Reply:
It’s fine